### PR TITLE
[docs] fix logging categories table markup in developer docs

### PR DESCRIPTION
**Pull request by mprins**
needs a newline.

currently https://docs.geotools.org/latest/developer/conventions/code/log.html#logging-messages
looks like this:

image

<Include a few sentences describing the overall goals for this Pull Request>

## Checklist

> Reviewing is a process done by project maintainers, mostly on a volunteer basis. We try to keep the overhead as small as possible and appreciate if you help us to do so by completing the following items. Feel free to ask in a comment if you have troubles with any of them.

For all pull requests:

- [x] Confirm you have read the [contribution guidelines](https://github.com/geotools/geotools/blob/master/CONTRIBUTING.md) 
- [ ] You have sent a Contribution Licence Agreement (CLA) as necessary (not required for small changes, e.g., fixing typos in documentation)
- [x] Make sure the first PR targets the master branch, eventual backports will be managed later. This can be ignored if the PR is fixing an issue that only happens in a specific branch, but not in newer ones.
- [x] The changes are not breaking the build in downstream projects using SNAPSHOT dependencies, GeoWebCache and GeoServer.

The following are required only for core and extension modules (they are welcomed, but not required, for unsupported modules):
- [ ] There is an issue in [Jira](https://osgeo-org.atlassian.net/projects/GEOT) describing the bug/task/new feature (a notable exemptions is, changes not visible to end users). The ticket is for the GeoTools project, if the issue was found elsewhere it's a good practice to link to the origin ticket/issue.
- [x] The pull request contains changes related to a single objective. If multiple focuses cannot be avoided, each one is in its own commit and has a separate ticket describing it.
- [ ] PR for bug fixes and small new features are presented as a single commit
- [ ] Commit message must be in the form "[GEOT-XYZW] Title of the Jira ticket"
- [ ] New unit tests have been added covering the changes
- [ ] This PR passes all existing unit tests (test results will be reported by travis-ci after opening this PR)
- [ ] This PR passes the [QA checks](https://docs.geotools.org/latest/developer/conventions/code/qa.html) (QA checks results will be reported by travis-ci after opening this PR)
- [x] Documentation has been updated accordingly.

Submitting the PR does not require you to check all items, but by the time it gets merged, they should be either satisfied or inapplicable.

### CODE REVIEW

File: docs/developer/conventions/code/log.rst [hunk 1]
```diff
@@ -156,4 +157,4 @@ Log4J offers similar functionalities with a wider range of handler implementatio
 
 We are not claiming that Java logging in superior to Log4J, neither we are forcing peoples to use Java logging. We push for usage of Java logging API, which may very well redirect to Log4J under the hood through java.util.logging.Log4JLoggerFactory implementations.
 
-Commons-logging is widely used in server containers, but other communities like scientists face a different picture. For example the NetCDF library developed by the University Corporation for Atmospheric Research (UCAR) uses SLF4J, yet other logging framework that aims to be a replacement for commons-logging.
\ No newline at end of file
+Commons-logging is widely used in server containers, but other communities like scientists face a different picture. For example the NetCDF library developed by the University Corporation for Atmospheric Research (UCAR) uses SLF4J, yet other logging framework that aims to be a replacement for commons-logging.
```
Comment (bradh) [inline docs/developer/conventions/code/log.rst]: Looks like you just did a newline at the end, although that should ready "yet another logging framework".

Comment (bradh) [reply docs/developer/conventions/code/log.rst]: Which it turns out I can fix.

Comment (mprins) [reply docs/developer/conventions/code/log.rst]: Not sure what happened there, maybe a newline correction by the online GH editor, my "fix" is the extra blank line 43, before the table starts

Comment (mprins) [reply docs/developer/conventions/code/log.rst]: so +1 for fixing another thing

**Pull request merged**